lib/macro.ts

```diff
@@ -48,8 +48,8 @@ const macroHandler: MacroHandler = (params) => {
 
   // Begin mutations
 
-  programPath.scope.rename(tsInterfaceCheckerIdentifier);
-  programPath.scope.rename(onceIdentifier);
+  programPath?.scope.rename(tsInterfaceCheckerIdentifier);
+  programPath?.scope.rename(onceIdentifier);
```

Comment:
Why do we need to add these "optional chaining" operators in this file?
With the optional chaining operators, if `programPath` is somehow `null | undefined` (which it _should_ never be), the calls to `.rename()` will be silently skipped, when we actually want an error instead (to let us know something went wrong)

Reply:
Sorry. I dont know why, but my Intellij idea says error is here. I'll reverted this.

Reply:
This hasn't changed yet with your updates. If you want to get rid of the TS errors you can use `!` instead of `?`, e.g.:
```suggestion
  programPath!.scope.rename(onceIdentifier);
```

and that doesn't change the compiled code.

I'm not sure why the TS error doesn't come up when building.. I guess the build is not using "strict null checks" (`--strictNullChecks`), whereas your IDE *is* using strict null checks for some reason?